governance-charter.md

```diff
@@ -232,3 +232,44 @@ majority vote of the governance committee members, but closed meetings do not
 count towards the required meeting frequency. The governance committee will
 inform the community if a particular meeting is closed, as well as the general
 reasons for this, prior to the meeting taking place.
+
+## Community Manager Role
+
+In order to facilitate end-user and contributor experience programs that cut
+across multiple SIGs in the OpenTelemetry project, the governance committee
+may appoint Community Managers (CMs) to act as liaisons, facilitators, and 
+organizers for such programs. The goal of this role is to ensure the health of
+the OpenTelemetry end-user and contributor community.
+
+### Community Manager Responsibilities
+
+- Act as stewards for the OpenTelemetry end-user and contributor community by
+  organizing events and coordinating programs.
+- Manage the OpenTelemetry social media and blog presence.
+- Work with the CNCF and member organizations on cross-functional projects as
+  needed.
+- Track and report on OpenTelemetry end-user and contributor community health
+  for the GC.
+- Grow the OpenTelemetry end-user and contributor community by advocating for
+  programs and initiatives that will attract new users or aid contributor
+  experience.
+  
+### Community Manager Eligibility, Term, and Selection
+
+CMs shall be nominated and approved by the GC for one-year terms on an as-needed 
+basis, by simple majority vote. The GC shall re-confirm CMs on an annual basis.
```

Comment:
> The GC shall re-confirm CMs on an annual basis.

Who is "watching the clock" on these? One idea would be to have the GC appoint CMs as-needed, then to reevaluate whether CMs are still needed (and/or are fulfilling the responsibilities) as part of the annual OTel election cadence.

I don't have a strong objection to this proposal, btw, I'm just imagining that the GC may simply forget when it's been 52w since any given CM was last confirmed in the role.

Reply:
Added clarifying language that the re-confirmation should happen at the first GC meeting after a GC election.